README.md

```diff
@@ -160,142 +161,194 @@ Structured Extraction is guided by the provided schema. We support Pydantic Mode
 * [Page Classification](https://docs.tensorlake.ai/document-ingestion/parsing/page-classification)
 * [Signature Detection](https://docs.tensorlake.ai/document-ingestion/parsing/signature)
 
-## Data Workflows
+## Build Durable Agentic Applications in Python
+
+Tensorlake Agentic Applications allow you to chain Python functions into durable, distributed workflows. These functions can be deployed to a serverless backend with a single command.
 
-Workflows enables building and deploying workflow APIs. The workflow APIs are exposed as HTTP Endpoints.Functions in workflows can do anything from calling a web service to loading a data model into a GPU to run inference.
+**No Queues**: We handle the state and orchestration.
 
-### Workflows Quickstart
+**Zero Infra**: Write Python, deploy to Tensorlake.
 
-Define a workflow by implementing its data transformation steps as Python functions decorated with `@tensorlake_function()`.
-Connect the outputs of a function to the inputs of another function using edges in a `Graph` object, which represents the full workflow.
+### Agentic Applications Quickstart
+
+Define an Application by implementing its data transformation steps as Python functions decorated with `@function()`. Specify the image needed for each function and any required secrets as parameters to the `@function()` decorator. Add the `@application()` decorator to each entrypoint function. 
```

Comment:
Write an Application in Python, decorate the entrypoint of your application with `@application()` and the functions with `@function()` if you want their state to be checkpointed or run them in sandboxes. Specify the image needed for functions and any required secrets as parameters to the  decorator.